docs/developers/guides/token.md

```diff
@@ -0,0 +1,198 @@
+# Launching a Token
+Welcome to our guide on launching a token collection using the Koinos Contract Standard (KCS-1) for Tokens and the AssemblyScript SDK for Koinos. In this tutorial, we'll walk you through the process of creating and deploying your own token on the Koinos blockchain. Whether you're a seasoned developer or new to blockchain development, this step-by-step guide will provide you with the knowledge and tools necessary to bring your token project to life. Let's dive in and explore the exciting world of token creation on Koinos!
+
+Before starting, ensure that you have already set up your Koinos AssemblyScript SDK environment by following [this guide](../as-sdk.md).
+
+---
+## Setting up the project
+To begin, clone the token repository and install the required dependencies.
+
+```sh
+git clone https://github.com/roaminro/koinos-sdk-as-examples/
+cd koinos-sdk-as-examples/token
+yarn install
+```
+
+---
+## Customizing your token
+Customize the specifics of your token project by modifying `token/assembly/Token.ts`. Define the following:
+
+- `_name`: The name of your token
+
+- `_symbol`: The symbol for your token
+
+- `_decimals`: The decimal places for your token
+
+
+```ts
+- _name: string = "Token";
+- _symbol: string = "TKN";
+- _decimals: u32 = 8;
+```
+
+After making your changes, update the token name and symbol in the unit test file located at `token/assembly/__tests__/token.spec.ts`.
+We will need to change the test for the `name` and `symbol` functions to reflect your token name and symbol.
+
+```ts
+  it("should get the name", () => {
+    const tkn = new Token();
+
+    const args = new token.name_arguments();
+    const res = tkn.name(args);
+
+    expect(res.value).toBe("[your token name]");
+  });
+
+  it("should get the symbol", () => {
+    const tkn = new Token();
+
+    const args = new token.symbol_arguments();
+    const res = tkn.symbol(args);
+
+    expect(res.value).toBe("[your token symbol]");
+  });
+```
+
+---
+## Building and testing
+Build and debug your project using the following command from the `assembly/token` directory:
+
+```sh
+yarn build:debug
+```
+
+Run unit tests to ensure your token is functioning as expected:
+
+```sh
+yarn test
+```
+
+---
+## Compiling the contract
+Once you've made all the necessary modifications and your tests are passing, you're ready to build the release version of your contract. From the `assembly/token` directory, run:
+
+```sh
+yarn build:release
+```
+
+After the build completes, locate your `.wasm` and `.abi` files:
+
+- `.wasm` file: `/koinos-sdk-as-examples/token/assembly/build/release/contract.wasm`
+- `.abi` file: `/koinos-sdk-as-examples/token/abi/token.abi`
+
+With these files, let's generate a Koinos wallet with `koinos-cli` and deploy your contract.
+
+---
+
+## Deploying the contract
+### The Koinos CLI
+
+If you haven't done so yet, download the Koinos CLI [here](https://github.com/koinos/koinos-cli/releases).
```

Comment:
Instead of pointing users to the release page, point them to the Koinos CLI page on the documentation site. This way if anything changes we don't have to update all areas of the documentation.